docs/sphinx/source/ReleaseNotes.md

```diff
@@ -20,7 +20,7 @@ Users performing online updates are encouraged to update from [4.0.559.4](#40559
 <h4> New Features </h4>
 
 * FRL respects PLAN_CACHE_*_MAX_ENTRIES - [PR #3156](https://github.com/FoundationDB/fdb-record-layer/pull/3156)
-<h4> Bug Fixes </h4>
+  <h4> Bug Fixes </h4>
```

Comment:
I'm not sure what happened here, but we probably want to back this out and take what's on `main`

Reply:
done